docs/man/rpm-man-template.scd

```diff
@@ -0,0 +1,83 @@
+RPMCOMMAND(1)
+
+# NAME
+rpmcmd - Example rpm man page
+
+# SYNOPSIS
+*rpmcmd* [options] [_ARGUMENT_] ...
+
+# DESCRIPTION
+Describe what *rpmcmd* command does in detail.
+
+Put distinct operational modes to their own subsections.
+
+Typographic conventions:
+- Use bold for literal values such as command name and options:
+*cmd* does X, with option *--do-this* does Y instead
+- Use underline uppercase to denote user arguments: _ARGUMENT_
+- Use underline for literal file names: _hello-1.0-1.noarch.rpm_
+- Use alphabetical order whenever multiple things are listed and
+  there's no contextual reason to do otherwise
+- Put the short variant first when listing option aliases
+- Put an empty line before a new section/option/etc
+- Put text immediately below a section/option/etc (ie without an empty line)
+- Put optional elements into [] brackets
+- Do not emphasize separators like ",", "=", "[]", "<>", "|" etc
+
+When in doubt, consult *man-pages*(7).
+
+# OPTIONS
+Describe common options and in this section.
+
+Typographic conventions:
+- Put option aliases on separate lines
+- Put option arguments with pre-determined values into <> (see below)
+
+*-d*
```

Comment:
Oh, this is missing a command after `-d` :smile: Got lost in all the noise.

Reply:
Yup. So many details, but by deity our own examples should be right, these are nits that *should* be picked to th bone :smile: